app/src/main/res/layout/fragment_transaction_add_2.xml

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:background="@color/content_main"
+        tools:context=".profile.ui.MyProfileFragment" >
+
+        <TextView
+            android:id="@+id/textView3"
```

Comment:
pls remove if not used else use snake case

Reply:
fix in 49df7d5e62794cf0a6698424f963f76f5828af96